flexmeasures/data/schemas/sensors.py

```diff
@@ -94,9 +94,12 @@ def _serialize(self, sensor: Sensor, attr, data, **kwargs) -> int:
 
 
 class QuantityOrSensor(MarshmallowClickMixin, fields.Field):
-    def __init__(self, to_unit: str, *args, **kwargs):
+    def __init__(
+        self, to_unit: str, default_src_unit: str | None = None, *args, **kwargs
+    ):
         super().__init__(*args, **kwargs)
         self.to_unit = ur.Quantity(to_unit)
+        self.default_src_unit = default_src_unit
```

Comment:
The purpose of this variable is to define a default unit for when someone is passing a variable as a numeric. For example, in the case of the efficiency, before we were assuming to be dimensionless. This could be used in some future to keep backwards compatibility of some fields.

For future reference, I've added a short explanation in the docstring.